src/hydra/addFilters.js

```diff
@@ -0,0 +1,39 @@
+import Filter from "../Filter";
+import fetchResource from "./fetchResource";
+
+export default api => {
+  const promises = [];
+
+  for (const resource of api.resources) {
+    let promise = fetchResource(resource.url).then(response => {
+      const resourceFilters = [];
+
+      if (response.filters) {
+        for (const filter of response.filters) {
+          let property = filter.property;
+
+          // TODO : To prevent PropertyFilter, maybe should be handle specifically ?
```

Comment:
what do you mean?

Reply:
The `PropertyFilter` provides `null` as `property`, but it can be applied to all properties. I thought i could iterate on each properties to create all filters, but this behavior could be different on another type of filter providing `null` as `property`, so i guess just skipping the filter might be what we want.